clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -268,7 +272,11 @@ public synchronized AbstractStorageEngine openStore(
       storeConfig.setStorePersistenceType(storeConfig.getPersistenceType());
     }
 
-    LOGGER.info("Creating/Opening Storage Engine {} with type: {}", topicName, storeConfig.getStorePersistenceType());
+    LOGGER.info(
+        "Creating/Opening Storage Engine {} with type: {} and trace {}",
+        topicName,
+        storeConfig.getStorePersistenceType(),
+        new Exception().getStackTrace());
```

Comment:
Is this stacktrace useful in a steady state?
Stacktrace normally would  lead people to think there are some issues with it.